src/WebJobs.Script/Utility.cs

```diff
@@ -94,6 +94,26 @@ internal static async Task DelayAsync(int timeoutSeconds, int pollingIntervalMil
             }
         }
 
+        /// <summary>
+        /// Delays while the specified condition remains true.
+        /// </summary>
+        /// <param name="timeoutSeconds">The maximum number of seconds to delay.</param>
+        /// <param name="pollingIntervalMilliseconds">The polling interval.</param>
```

Comment:
There's a lot of duplication here (this is pretty much the same code as the delay above), you can consolidate these methods into one that will wrap the parameter that doesn't return a `Task`

Reply:
done!